### DISCUSSION TITLE
Hi, for some reason the permissions setting Im using for Wireless LAN is not working.

### DISCUSSION THREAD

**Discussion by 33354** [netbox-community/netbox#8663] [Q&A]
### NetBox version

v3.1.5

### Python version

3.8

### Steps to Reproduce

1. I set the Object Type to Wireless > Wireless LAN
2. set action to view only
3. assigned the correct user group

### Expected Behavior

user should only be able to view

### Observed Behavior

User is still able to add, delete, change in Wireless LAn

**Comment by candlerb** [comment]
And they definitely don't have permissions assigned via some other group, or the Superuser flag enabled on their account?
